openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1920,6 +1920,62 @@ The following is a non-normative example of a Credential Request with Credential
 
 The value of the `credential` claim in the Credential Response MUST be a string that is the base64url-encoded representation of the issued Credential.
 
+## IETF SD-JWT VC
+
+This section defines a Credential Format Profile for Credentials complying with [@!I-D.ietf-oauth-sd-jwt-vc].
+
+### Format Identifier
+
+The Credential format identifier is `vc+sd-jwt`.
+
+### Credential Issuer Metadata {#server_metadata_vc_sd-jwt}
+
+The following additional Credential Issuer metadata are defined for this Credential format to be added to the `credentials_supported` parameter in addition to those defined in (#credential-issuer-parameters).
+
+* `credential_definition`: REQUIRED. JSON object containing the detailed description of the Credential type. It consists of the following sub elements:
```

Comment:
Not sure why `credential_definition` is needed. I'd remove it.

Reply:
`credential_definition` was introduced in w3c vcdm examples because of the collision btw `type` claim in VCI and VCDM. there is no such collision in sd-jwt vc (there is `vct` claim instead) so if there is a broader agreement, happy to change, but would like to see more support in this direction before making a change because it is not a small change.

Reply:
`credential_definition` is unnecessary bloat. We should remove it.

Reply:
I agree it complicates the SD JWT VC profile in a way I dont see we get any value from.

Reply:
As per chat with Kristina/Daniel/Paul/Christian we thought we might as well remove credential_definition before this is merged.

Reply:
removed